GoogleForCreators/web-stories-wp
# Design System: More colors for the theme

## Summary

[Figma's colors in the new design system ](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3447%3A64541) were updated after I initially set up the structure. This PR is to update the theme to use the finalized colors. It also updates the theme values used in existing components of the design system (button, tab, dialog)

## Relevant Technical Choices

- the buttons now share active and disabled colors regardless of variety so i consolidated those. 
- the designs have no focus, active, hover states for tabs so i removed that and added a light theme variety to see both. 

## To-do



## User-facing changes

None, only storybook

## Testing Instructions

- Spin up storybook and go to designSystem/colors - see that there's colors added, they should match the values in figma. 
- Updated colors for button, tab, dialog (the overlay) - see that these are consistent with figma's updates. 
- The dark theme values should be set as "inverted" values when in light mode and the light theme values should be set as "inverted" values when in dark mode. This will help for components that use the opposite theme colors as the mode it's set in which is a new update this week.


---



Fixes #
